includes/class-sensei.php

```diff
@@ -336,6 +336,8 @@ private function __construct( $main_plugin_file_name, $args ) {
 		 * @param {Sensei_Main} $sensei Sensei object.
 		 */
 		do_action( 'sensei_loaded', $this );
+
+		new Sensei_Email_Sender();
```

Comment:
Would it make more sense to add this in `initialize_global_objects` where the other classes are being instantiated?